#Edge information published from image processor
#Line colors in image
string[] colors

#Position pairs for all bands
int16[] pos_avgs
